pkg/services/featuremgmt/manager_test.go

```diff
@@ -0,0 +1,31 @@
+package featuremgmt
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFeatureToggleSetup(t *testing.T) {
+	ft := WithFeatures("a", "b", "c")
+	assert.True(t, ft.IsEnabled("a"))
```

Comment:
I prefer using `github.com/stretchr/testify/require` instead of `assert` since in most cases first error is what we aiming for when looking at test failure, the following errors in test are usually similar or caused by the first one etc - i.e. not really deterministic after first one.